.github/workflows/.abc-checks.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 The Authors (see AUTHORS file)
```

Comment:
@dcreey I'm confused. This is a "dot" file, so it's only supposed to apply to this repo. Why is it named "abc-checks"?